v1/middleware/auth.go

```diff
@@ -124,6 +124,37 @@ func isBanned(ctx *gin.Context) (bool, error) {
 	return true, nil
 }
 
+func isSuspended(ctx *gin.Context, service string) gin.HandlerFunc {
+	return func(ctx *gin.Context) {
+		client, err := jwt.ExtractClient(ctx.GetHeader("Authorization"))
+		if err != nil {
+			rest.ResponseMessage(ctx, http.StatusUnauthorized)
+			ctx.Abort()
+			return
+		}
```

Comment:
```suggestion
		client, _ := jwt.ExtractClient(ctx.GetHeader("Authorization"))
```
ini bisa dihapus aja, pengecekan authorization bisa lewat `middleware.Auth()`